XenModel/Actions/Updates/DownloadAndUpdateClientAction.cs

```diff
@@ -31,9 +31,11 @@
 

```

Comment:
nit: the class should probably be in the `XenAdmin.Actions.Updates` namespace. I know this is wildly inconsistent in the rest of the codebase, but it'd be nice to start keeping track of this when editing files

The same applies for `ClientUpdateAlert.cs`

Reply:
The action I agree with, the alert not so much as it is not an action. It is also intertwinned with many more classes and dont want to break it.